### ISSUE TITLE
Test Tour: Create a mobile app using angular, sass (JavaScript)

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode#3715]
http://ionicframework.com/

**Comment by stevencl** [comment]
Some comments on the experience

- I was pleasantly surprised when I got autocomplete for ionic elements inside an html file. I didn't do anything special to set that up. But it set up some expectation that I might get intellisense and autocomplete in my javascript too, which wasn't the case.
- When I opened a js file, I saw a green lightbulb in the status bar which indicated that I should create a jsconfig.json to get richer intellisense. Clicking on it opened a prompt at the top of the window with a button to create a jsconfig.json file. I clicked it and it created the new file for me. However, the new file replaced the file that I had been looking at in the editor meaning I had to go back to the explorer to find the file. We should consider opening newly generated files inside another editor if one is available. Opened bug https://github.com/Microsoft/vscode/issues/4499
- I went back to the file and started typing some angular code. I didn't get intellisense although the previous step I took implied that I might get intellisense. Perhaps the jsconfig.json file should include a comment directing people to install the typings file for whatever frameworks they are using. Opened bug: https://github.com/Microsoft/vscode/issues/4498
- After tsd init and tsd install angularjs/ I have great intellisense in my js
- Whenever I specify the ng-controller directive inside an html element I do not get any autocomplete for the name of the controller. In Atom, I get string autocompletion as soon as I start typing the name of the controller. This is very helpful. Opened bug: https://github.com/Microsoft/vscode/issues/4500
